src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressElement.java

```diff
@@ -129,14 +129,12 @@ public void setColumns(String columns) {
     @Override
     public boolean accept(AuditEvent event) {
         // reject if file or check module mismatch?
-        if (isFileNameAndModuleNotMatching(event)) {
-            return true;
-        }
+        return isFileNameAndModuleNotMatching(event)
+                // reject if no line/column matching
+                || (lineFilter != null || columnFilter != null)
```

Comment:
please remove comments.
from second block `|| (lineFilter != null .....` please create function with clear name.

Big/complicated return statements are not good, that is why original author used separate return. Now code become hard to read -so extra refactoring is requied.